src/App.tsx

```diff
@@ -1,14 +1,32 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
-import { Loader } from './components/Loader';
+// import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState(():Todo[] => []);
+  const [todoId, selectTodo] = useState(0);
+  const [visibleTodos, getVisibleTodos] = useState(todos);
+
+  const selectedTodo = todos.find(todo => todo.id === todoId);
+
+  const loadData = (async () => {
```

Comment:
The loadData function should be wrapped in useCallback to avoid creating a new function on every render. Example: `const loadData = useCallback(async () => { ... }, []);` 🧙